OWNERS_ALIASES

```diff
@@ -264,28 +264,6 @@ aliases:
     - srm09
   # end sigs.k8s.io/cluster-api-provider-vsphere/OWNERS_ALIASES
 
-  # mostly copied from sigs.k8s.io/controller-runtime/OWNERS_ALIASES
```

Comment:
I inlined the owner aliases as we're then able to update the approvers/reviewers ourselves without requiring top-level approval